src/components/App.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ButtonPanel from './buttonPanel';
+import Display from './display';
+
+function App() {
+  return (
+    <div>
```

Comment:
- [ ] Has children components wrapped in [React.Fragment](https://reactjs.org/docs/fragments.html#short-syntax) short syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896092/)) (Instead of using the div tag, please use the React fragment one to comply with the project requirements.)